kedro/framework/project/__init__.py

```diff
@@ -216,13 +216,36 @@ def configure(self, pipelines_module: str | None = None) -> None:
 class _ProjectLogging(UserDict):
     def __init__(self) -> None:
         """Initialise project logging. The path to logging configuration is given in
-        environment variable KEDRO_LOGGING_CONFIG (defaults to default_logging.yml)."""
-        path = os.environ.get(
-            "KEDRO_LOGGING_CONFIG", Path(__file__).parent / "default_logging.yml"
-        )
+        environment variable KEDRO_LOGGING_CONFIG (defaults to conf/logging.yml)."""
+
+        # Check if a user path is set in the environment variable
+        user_logging_path = os.environ.get("KEDRO_LOGGING_CONFIG")
+
+        # Check if the default logging configuration exists
+        default_logging_path = Path("conf/logging.yml")
+        if not default_logging_path.exists():
+            default_logging_path = Path(__file__).parent / "default_logging.yml"
+
+        # Use the user path if available, otherwise, use the default path
+        if user_logging_path and Path(user_logging_path).exists():
+            path = Path(user_logging_path)
+        else:
+            path = default_logging_path
+
+        # Load and apply the logging configuration
         logging_config = Path(path).read_text(encoding="utf-8")
         self.configure(yaml.safe_load(logging_config))
 
+        # Log info about the logging configuration
+        if not user_logging_path and path.resolve() == (
+            Path("conf/logging.yml").resolve()
```

Comment:
@SajidAlamQB I suspect the windows test is failing because on windows this path is not being resolved properly, it happens on GHA. try absolute path - `(project_path / Path("conf/logging.yml")).resolve()`